README.rst

```diff
@@ -24,19 +24,19 @@ repository:
     **Deprecated**
     Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7.
 
-`readthedocs/build:5.0`
+`readthedocs/build:6.0`
     ``stable``
-    Ubuntu 18.04 supporting Python 2.7, 3.6, 3.7 and pypy3.5-7.0.0.
+    Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8 and PyPy3.5-7.0.0.
     This is the **stable** image supported by Read the Docs.
 
-`readthedocs/build:6.0`
+`readthedocs/build:7.0`
     ``latest``
     Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8 and PyPy3.5-7.0.0.
     This is the **latest** default image used for documentation builds and supported by Read the Docs.
 
-`readthedocs/build:7.0`
+`readthedocs/build:8.0`
     ``testing``
-    Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8 and PyPy3.5-7.0.0.
+    Ubuntu 20.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8, 3.9.0rc1 and PyPy3.5-7.0.0.
     For internal development **testing** only, not available for public usage yet.
```

Comment:
Isn't this available as a `testing` image on RTD? We just don't document it?

Reply:
Yes. This image is available via the config file `build: image: testing` but we don't document it.